Python/Iniciante/1178.py

```diff
@@ -0,0 +1,20 @@
+#Problema 1178 - Beecrowd - Iniciante - Nível 2
+
+#Leitura do valor de x
+x = float(input())
+
+#Declaraçao do vetor N
+N = []
+
+#Criação de um loop que preencherá a matriz com os 100 elementos
+for i in range(100):
+
+    #Acréscimo do valor de x no final o vetor N
+    N.append(x)
+    #Divisão do valor de x por dois para que o próximo elementa de N receba metade do valor atual de x
+    x = x/2
+
+#Criação de um loop para printar os elementos de N
+for i in range(100):
+    #uso do truncamento da quarta casa decimal 
+    print(f"N[{i}] = {N[i]:.4f}")
```

Comment:
Algumas questões para ficar mais rápido e didático o algoritmo:
- Não precisa fazer dois for, printa no mesmo for de preenchimento do vetor.
- Não precisa inicializar o vetor vazio, já coloca o "x" dentro e printa ele 😄.

```suggestion
#Colocando o primeiro valor no vetor
N = [x]

#Mostrando o primeiro valor do vetor
print(f"N[0] = {N[0]:.4f}") 

#Criação de um loop que preencherá o vetor com os 99 elementos restantes
for i in range(1,100):

    #Divisão do valor de x por dois para que o próximo elementa de N receba metade do valor atual de x
    x = x/2
    #Acréscimo do valor de x no final o vetor N
    N.append(x)
    #Printando os valores do vetor.
    print(f"N[{i}] = {N[i]:.4f}") 
```